namespaces/live-1.cloud-platform.service.justice.gov.uk/laa-cla-backend-staging/resources/rds.tf

```diff
@@ -79,12 +79,14 @@ resource "kubernetes_secret" "cla_backend_rds" {
   }
 
   data = {
-    endpoint = module.cla_backend_rds.rds_instance_endpoint
-    host     = module.cla_backend_rds.rds_instance_address
-    port     = module.cla_backend_rds.rds_instance_port
-    name     = module.cla_backend_rds.database_name
-    user     = module.cla_backend_rds.database_username
-    password = module.cla_backend_rds.database_password
+    endpoint         = module.cla_backend_rds.rds_instance_endpoint
+    host             = module.cla_backend_rds.rds_instance_address
+    port             = module.cla_backend_rds.rds_instance_port
+    name             = module.cla_backend_rds.database_name
+    user             = module.cla_backend_rds.database_username
+    password         = module.cla_backend_rds.database_password
+    replica_host     = module.cla_backend_replica.rds_instance_address
+    replica_endpoint = module.cla_backend_replica.rds_instance_address
```

Comment:
@said-moj , would this not be rds_instance_endpoint?

Reply:
@poornima-krishnasamy good catch, updated